HAZIK/MMárk/Mark_hazi3.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[27]:
+
+
+import matplotlib
+import networkx as nx
+import urllib
+import csv
+import matplotlib.pyplot as plt
+import statistics
+import numpy as np
+
+
+# In[68]:
+
+
+def generate_hazi3():
+        g = nx.read_edgelist("CA-GrQc.txt",create_using=nx.DiGraph(), nodetype = int)
+
+        print('Csomópontok száma:', g.number_of_nodes())
+        print('Klaszterezettségi együttható:', nx.average_clustering(g))
+        degrees = [len(list(g.neighbors(n))) for n in g.nodes()]
+        print('Átlagos fokszám:', sum(degrees)/len(degrees))
+        if nx.is_connected(nx.to_undirected(g)) == False:
```

Comment:
Ez így nem jó

Egyrészt nincs true ága a dolognak, mi történik ha connected a gráf?

Másrészt ilyet sose szabad h `if valami == False`, helyette `if not valami`